src/events/eventActions.js

```diff
@@ -43,12 +44,16 @@ export const startEventPolling = (queueId: number, eventId: number) => async (
     }
 
     const actions = [];
+    let startTypingNotificationLoop = false;
     for (const event of response.events) {
       lastEventId = Math.max(lastEventId, event.id);
       eventMiddleware(getState(), event);
       const action = eventToAction(getState(), event);
 
       if (action) {
+        startTypingNotificationLoop = !!(
```

Comment:
The `!!` is not necessary here. It is used to convert to 'boolean' and we already have that.